CHANGELOG.md

```diff
@@ -76,6 +76,9 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - `exonum_derive` crate has been added with custom derives for `ProtobufConvert`
   and `TransactionSet`. (#1055)
 
+- `TransactionResult` now serialized using protobuf. Empty description of the result
+  now equivalent to no description of the result. (#1075)
```

Comment:
Is really grammar correct here ?

Reply:
Some verbs are missing. I suggest changing the sentence to the following:
`TransactionResult` is now serialized using Protobuf. Empty description of the result is 
  now the equivalent of there being no description of the result.